mergesync/resources/schemas/dbscripts/postgresql/mergesync-0.05-0.06.sql

```diff
@@ -77,7 +78,6 @@ INSERT INTO mergesync.testnamemapping (servicename, mergetestname, automaticresu
 INSERT INTO mergesync.testnamemapping (servicename, mergetestname, automaticresults) VALUES ('Urinalysis', 'URINE', false);
 INSERT INTO mergesync.testnamemapping (servicename, mergetestname, automaticresults) VALUES ('Yersinia culture', 'YERSNA', false);
 
-TRUNCATE TABLE mergesync.mergetolkmapping;
```

Comment:
I'm not understanding why this line is getting deleted. In the normal bootstrap scenario, mergesync-0.04-0.05.sql will have inserted rows.

Reply:
The table was dropped and recreated just before this INSERT chunk, so the TRUNCATE is a no-op. Not causing any harm, of course.